AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
     }
-
+    
     enum ChooseSessionTypeView {
-        static let title: String = LocalizedStringKey("Let's begin").stringKey
-        static let message: String = LocalizedStringKey("How would you like to add your session?").stringKey
-        static let recordNew: String = LocalizedStringKey("Record a new session").stringKey
-        static let moreInfo: String = LocalizedStringKey("more info").stringKey
-        static let fixedLabel_1: String = LocalizedStringKey("Fixed session").stringKey
-        static let fixedLabel_2: String = LocalizedStringKey("for measuring in one place").stringKey
-        static let mobileLabel_1: String = LocalizedStringKey("Mobile session").stringKey
-        static let mobileLabel_2: String = LocalizedStringKey("for moving\naround").stringKey
-        static let orLabel: String = LocalizedStringKey("or").stringKey
-        static let syncTitle: String = LocalizedStringKey("Sync data from \nAirBeam3").stringKey
-        static let syncDescription: String = LocalizedStringKey("if you recorded with AirBeam3").stringKey
-
-    }
-
+        static let title: String = NSLocalizedString("Let's begin",
+                                                     comment: "")
+        static let message: String = NSLocalizedString("How would you like to add your session?",
+                                                       comment: "")
+        static let recordNew: String = NSLocalizedString("Record a new session",
+                                                         comment: "")
+        static let moreInfo: String = NSLocalizedString("more info",
+                                                        comment: "")
+        static let fixedLabel_1: String = NSLocalizedString("Fixed session",
+                                                            comment: "It consists of few parts. Please consider them together. Whole text is following: Fixed session for measuring in one place")
+        static let fixedLabel_2: String = NSLocalizedString("for measuring in one place",
+                                                            comment: "")
+        static let mobileLabel_1: String = NSLocalizedString("Mobile session",
+                                                             comment: "It consists of few parts. Please consider them together. Whole text is following: Mobile session for moving around")
+        static let mobileLabel_2: String = NSLocalizedString("for moving around",
+                                                             comment: "")
+        static let orLabel: String = NSLocalizedString("or",
+                                                       comment: "")
+        static let syncTitle: String = NSLocalizedString("Sync data from AirBeam3",
+                                                         comment: "")
+        static let syncDescription: String = NSLocalizedString("if you recorded with AirBeam3",
```

Comment:
all multipart strings here should be merged into single ones using techniques described above